lib/rubocop/cop/lint/void.rb

```diff
@@ -104,7 +104,7 @@ def check_var(node)
         end
 
         def check_literal(node)
-          return if !node.literal? || node.xstr_type?
+          return if !node.literal? || node.xstr_type? || node.range_type?
```

Comment:
I wonder why `literal?` doesn't cover them. I think they weren't immutable or something like this, but my memory on the subject is fuzzy.